test/TritonGPU/amd/accelerate-amd-matmul-mfma-gfx950.mlir

```diff
@@ -93,9 +93,7 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 4 : i32, ttg.targ
       %arg2: tensor<128x128x!tt.ptr<f32>, #blocked1>
       ) {
     // CHECK-NOT: tt.fp_to_fp
-    // CHECK-DAG: %[[CST0:.+]] = arith.constant dense<127> : tensor<128x4xi8, #linear>
-    // CHECK-DAG: %[[CST1:.+]] = arith.constant dense<127> : tensor<128x4xi8, #linear1>
-    // CHECK: tt.dot_scaled {{.*}} scale %[[CST1]], {{.*}} scale %[[CST0]], {{.*}} lhs = e2m1 rhs = e2m1
+    // CHECK: tt.dot_scaled {{.*}}, {{.*}}, {{.*}} lhs = e2m1 rhs = e2m1
```

Comment:
This could match the old IR with `scale`s? Consider making the regexpr explicit like `%{{[^ ]+}}` or something.